content-repo/extra-docs/releases/20.11.0.md

```diff
@@ -0,0 +1,1015 @@
+
+---
+
+### Get License ID Pack v1.0.1
+#### Scripts
+##### GetLicenseID  
+- Fixed Context output to match the outputs definition of: *License.ID*.
+- Updated the Docker image to: *demisto/python3:3.8.6.12176*i'm .
+
+---
+
+### Gmail Pack v1.1.0
+#### Integrations
+##### Gmail  
+- Added the ***gmail-send-as-add*** and ***gmail-forwarding-address-add*** commands.
+- Added new arguments support in the ***gmail-set-autoreply*** command.
+- Updated the Docker image to: *demisto/google-api:1.0.0.12690*.
+
+---
+
+### HelloWorld Pack v1.1.12
+#### Integrations
+##### HelloWorld  
+The **ip**, **domain**, and **helloworld-scan-results** commands were changed to return multiple entries (entry per indicator) instead of a single entry.
+
+---
+
+### IBM QRadar Pack v1.1.7
+#### Integrations
+##### IBM QRadar v2  
+Reverted the integration cache handling changes made in version 1.1.4.
+Added a new field to fetched incidents - **LinkToOffense** - link to the offense in the QRadar console.
+- Fixed an issue where the *source* argument did not work when passing more than one value to the **qradar-update-reference-set-value** command.
+
+##### IBM QRadar  
+Fixed an issue where the *source* argument did not work when passing more than one value to the **qradar-update-reference-set-value** command.
+
+---
+
+### Integrations & Incidents Health Check Pack v1.1.5
+#### Playbooks
+##### JOB - Integrations and Playbooks Health Check  
+Fixed report name
+
+#### Scripts
+##### GetFailedTasks  
+- Maintenance and stability enhancements.
+- Updated docker image to the latest version.
+
+---
+
+### MITRE ATT&CK Pack v1.1.6
+#### Integrations
+##### MITRE ATT&CK Feed  
+- Fixed an issue where the *mitrecreated* and the *mitremodified* fields were not populated correctly. 
+- Upgraded the Docker image to demisto/taxii2:1.0.0.12410.
+
+---
+
+### Machine Learning Pack v1.2.1
+#### Scripts
+##### HashIncidentsFields  
+Fixed an issue where custom fields were returned by default as part of the incident object.
+
+---
+
+### Manage Engine Service Desk Plus Pack v1.2.1
+#### Integrations
+##### Service Desk Plus  
+- Updated *assign* and *close* API endpoints because the previous ones scheduled to be deprecated on November 15th, 2020.
+- Updateed the Docker image to: *demistoq/python3:3.8.6.12176*.
+
+---
+
+### Micro Focus Service Manager Pack v1.0.2
+#### Integrations
+##### Micro Focus Service Manager  
+Added the *customFields* argument to the ***hpsm-create-incident*** command.
+
+---
+
+### Microsoft Cloud App Security Pack v1.0.9
+#### Integrations
+##### Microsoft Cloud App Security  
+Documentation and metadata improvements.
+- Fixed an issue where **fetch-incident** ignored the *First fetch time* integration parameter, and fetched the same incident multiple times.
+- **Breaking Change** microsoft-cas-activities-list command changed to return multiple entries (entry per indicator) instead of a single entry.
```

Comment:
@glicht we have a few breaking changes related to how indicators are returned, do we want to display them in the release notes in the product?